Rakefile

```
@@ -679,14 +669,9 @@ def display_prompt_response
   response == 'Y'
 end
 
-def fold(label)
-  puts "travis_fold:start:#{label}" if is_travis?
+# FIXME: This used to add Travis folding formatting, but we no longer use Travis. I'm leaving it here for the moment, but I think we should remove it.
+def fold(_)
   yield
```

Or, alternatively, keep it but change the implementation to use [Buildkite's equivalent feature](https://buildkite.com/docs/pipelines/managing-log-output#collapsing-output) 🙃 

```suggestion
def fold(label)
  puts "--- #{label}" if is_ci?
  yield
```